docs/manual/java/guide/production/ProductionOverview.md

```diff
@@ -14,33 +12,35 @@ The production environment determines the methods for packaging your services, m
+    * You can also use `sbt-native-packager` to build and publish [Docker images](https://developer.lightbend.com/docs/lightbend-orchestration/current/building.html) that can be used with Kubernetes, DC/OS, or any other Docker-compatible production environment.
 
     * Otherwise, you can configure `sbt-native-packager` to produce [other archive formats](https://www.scala-sbt.org/sbt-native-packager/formats/universal.html#build), [Windows (MSI) installers](https://www.scala-sbt.org/sbt-native-packager/formats/windows.html), Linux packages in [Debian](https://www.scala-sbt.org/sbt-native-packager/formats/debian.html) or [RPM](https://www.scala-sbt.org/sbt-native-packager/formats/rpm.html) format and more.
 
 * Maven has a variety of plugins to produce artifacts for various platforms:
 
     * The [Maven Assembly Plugin](http://maven.apache.org/plugins/maven-assembly-plugin/) can be used to build archives in zip, tar and other similar formats.
 
-    * The [Docker Maven Plugin](https://dmp.fabric8.io/) is recommended for building Docker images. Maven support in Lightbend Orchestration is currently under development.
+    * The [Docker Maven Plugin](https://dmp.fabric8.io/) is recommended for building Docker images.
 
     * Other Maven packaging plugins in varying states of maturity are available from the [MojoHaus Project](https://www.mojohaus.org/plugins.html).
 
 * At runtime, services need to locate the addresses of other services to communicate with them. This requires you to configure an implementation of a [`ServiceLocator`](api/index.html?com/lightbend/lagom/javadsl/api/ServiceLocator.html) that Lagom uses to look up the addresses of services by their names. The production environment you choose might provide a service discovery mechanism that you can use with Lagom.
 
-    * For simple deployments, Lagom includes a built-in service locator that uses addresses specified in the service configuration ([described below](#Using-static-values-for-services-and-Cassandra)).
+    * For simple deployments, Lagom includes a built-in service locator that uses hardcoded addresses specified in the service configuration ([described below](#Using-static-values-for-services-and-Cassandra)).
 
-    * Lightbend Orchestration provides an open-source [`ServiceLocator` implementation](https://developer.lightbend.com/docs/lightbend-orchestration/current/features/service-location.html) that integrates with the service discovery features of Kubernetes or DC/OS, or any other environment that supports service discovery via DNS.
+    * When using dynamic deployments where processes don't run on static IPs, you can use the open-source [`lagom-akka-discovery-service-locator`](https://github.com/lagom/lagom-akka-discovery-service-locator)  that can integrate with the service discovery features of Kubernetes or DC/OS, or any other environment supported by [Akka's Service Discovery](https://doc.akka.io/docs/akka/2.5/discovery/index.html).
```

Comment:
Maybe we should just recommend `lagom-akka-discovery-service-locator` in every case, since it also has a config implementation and can be extended to integrate with others.